pom.xml

```diff
@@ -46,35 +46,11 @@
 		<sonar.organization>camel-tooling</sonar.organization>
 		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
-		<version.camel>3.18.0-SNAPSHOT</version.camel>
+		<version.camel>3.18.0</version.camel>
 		<!-- must be aligned with the one from Camel -->
 		<version.jaxb>2.3.3</version.jaxb>
 		<version.junit.pioneer>1.7.1</version.junit.pioneer>
 	</properties>
-	<repositories>
```

Comment:
Why was this removed?

Reply:
it was added to access the 3.18.0-SNAPSHOT Camerl version awaiting for the official release. Some features were requiring it. usually we try to stay on released version so no need of the snapshot repository. The other repository is the genric Maven repository. it was provided with the Apache snapshots to avoid performance hiccups withthe Apache Snapshot maven repository.